packages/pilot/src/pages/Account/actions/epic.js

```diff
@@ -133,6 +133,27 @@ const companyEpic = (action$, state$) => action$.pipe(
         payload: errorPayload,
       }))
   }),
+  mergeMap((action) => {
+    if (action.error) {
+      return action
+    }
+
+    const { value: state } = state$
+    const { account: { client } } = state
+
+    return client
+      .transactions
+      .search({ count: 1 })
+      .then(({ result }) => {
```

Comment:
Como vc está só usando o `count`  vc pode fazer assim:
```suggestion
      .then(({ result: { total: count } = { } }) => {
```

Reply:
Fica melhor msm, ajustado :heavy_check_mark: :smile: